SecretSanta/test/SecretSanta.Api.Tests/Controllers/GroupsControllerTests.cs

```diff
@@ -262,13 +263,10 @@ public async Task CreateAssignments_WithError_ReturnsBadRequest()
             HttpClient client = factory.CreateClient();
 
             //Act
-            HttpResponseMessage response = await client.PutAsJsonAsync("/api/groups/42/assign", new Dto.UpdateGroup
-            {
-                Name = "Changed"
-            });
+            HttpResponseMessage response = await client.PostAsJsonAsync("/api/groups/42/assign", 42);
```

Comment:
nit: Your post method does not read anything from the body, so you don't need to pass a value. You can just pass nothing.
```suggestion
            HttpResponseMessage response = await client.PostAsJsonAsync("/api/groups/42/assign", "");
```